portfolio/src/main/webapp/game.js

```diff
@@ -100,10 +117,106 @@ class Board {
       } else {
         this.board[row][col] = 2;
       }
+      this.numTiles++;
+      this.checkGameState(this.redTrue);
       this.redTrue = !this.redTrue;
+    } else {
+        alert("Cannot place tile there. Try again.");
+    }
+  }
+  
+  /*
+  * Checks whether move made by redTrue player resulted in win or if game board is full.
+  * Sends alert with win message or gameOver message if board is full and allows user to
+  * restart game.  
+  */
+  checkGameState(redTrue) {
+    var player = 0;
```

Comment:
Consider using ES6 let/const instead of var here and everywhere:
https://hacks.mozilla.org/2015/07/es6-in-depth-let-and-const/